### ISSUE TITLE
Add support for other (non-V8) trace formats

### ISSUE THREAD

**Issue by mourner** [mapbox/flamebearer#3]
In theory, it shouldn't be much of a trouble to add support for other trace formats. The common ones I've seen are:

- [ ] Perf traces (`perf report`)
- [ ] DTrace ticks
- [ ] FlameGraph folded traces (BCC, FlameGraph `stackcollapse`)
- [ ] Perf folded traces (`perf report -g folded` on Linux v4.5+)

cc @springmeyer

**Comment by rebornix** [comment]
I'll suggest to support the output of Chrome Dev Tools JavaScript Profiling.

**Comment by mourner** [comment]
@rebornix good idea, added to the list. Code for parsing it (for reference): https://github.com/bengl/cpuprofile2stackcollapse/blob/master/index.js

**Comment by mourner** [comment]
No longer relevant: #16
